benchmarking_hallucination_metrics/benchmark_hallucination_metrics.ipynb

```diff
@@ -0,0 +1,4614 @@
+{
+  "cells": [
+    {
+      "cell_type": "markdown",
+      "metadata": {
+        "id": "Gdq2ZGSlY1wH"
+      },
+      "source": [
+        "# Benchmarking Hallucination Detection Methods\n",
+        "\n",
+        "In this notebook, we will benchmark hallucination detection methods on subsets of the HaluEval dataset.\n",
+        "\n",
+        "Please see our related results paper at [link todo]\n",
```

Comment:
I think I'll drop this unless we get our blog out simultaneously, but that seems difficult!